src/ResourceManager/Network/Commands.Network/VirtualNetwork/NewAzureVirtualNetworkCommand.cs

```diff
@@ -76,12 +76,18 @@ public class NewAzureVirtualNetworkCommand : VirtualNetworkBaseCmdlet
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "A hashtable which represents resource tags.")]
         public Hashtable Tag { get; set; }
-        
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
@murilogr nit: you can remove the `ValueFromPipelineByPropertyName` property here